packages/react/dialog/src/Dialog.stories.tsx

```diff
@@ -210,15 +210,16 @@ export const Animated = () => (
 export const ForcedMount = () => (
   <Dialog>
     <DialogTrigger>open</DialogTrigger>
-    <DialogPortal>
-      <DialogOverlay className={overlayClass} forceMount />
-      <DialogContent className={contentDefaultClass} forceMount>
+    <DialogPortal forceMount>
+      <DialogOverlay className={overlayClass} />
+      <DialogContent className={contentDefaultClass}>
         <DialogTitle>Title</DialogTitle>
         <DialogClose>close</DialogClose>
       </DialogContent>
     </DialogPortal>
   </Dialog>
 );
+ForcedMount.parameters = { chromatic: { disable: false } };
```

Comment:
What's the reason for enabling this story separately as opposed to adding a new thing in the Chromatic story as we usually do?

Reply:
The dialog chromatic story is a PITA 😅  it's a bunch of absolutely positioned dialogs with 20% width so adding a 5th was a lot of manually fiddling hardcoded values to get things positioned correctly and just made things hard to visualise with 5 cramped dialogs in a row.

I contemplated updating the whole story to take advantage of the new `<Portal container={} />` API but that would have made the visual changes in this PR trickier to review. Happy to give it a go though if preferred?

Reply:
Ah yes, of course, I forgot about that. That's fine then, I think at some point we could split back into stories given we have quite a lot of snapshots allowance.